### PR TITLE
Feature/optimize tomcat

### PR DESCRIPTION
**Pull request by cesarjorgemartinez**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,9 @@ RUN yum update -y && yum install -y wget \
   && echo "action.url = http://${PERSEO_FE_URL}/actions/do" >>  /etc/perseo-core.properties \
   && echo "" >>  /etc/perseo-core.properties \
   && echo "# Time in milliseconds (long) to \"expire\" a \"dangling\" rule" >>  /etc/perseo-core.properties \
-  && echo "rule.max_age= 60000" >>  /etc/perseo-core.properties
+  && echo "rule.max_age= 60000" >>  /etc/perseo-core.properties \
+  # Optimize Tomcat optimize
```
Comment (fgalan) [inline Dockerfile]: Optimize Tomact optimize? :)

I guess the second "optimize" should be removed

Comment (cesarjorgemartinez) [inline Dockerfile]: Fixed :-)

**Comment by fgalan** [comment]
LGTM

**Comment by AlvaroVega** [comment]
LGTM

**Pull request merged**